scripts/translation/xliffmerger.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+import xml.etree.ElementTree as ET
+import os
+import logging
+
+logging.basicConfig(level=logging.INFO)
+
+
+def delete_trans_units(body_element, trans_units, units_to_remove):
+    to_delete = [tr_unit for tr_unit in trans_units if tr_unit.attrib['id'] in units_to_remove]
+    for item in to_delete:
+        body_element.remove(item)
+
+
+def add_trans_units(body_element, trans_units, units_to_add):
+    to_add = [tr_unit for tr_unit in trans_units if tr_unit.attrib['id'] in units_to_add]
+    for item in to_add:
+        item.set('state', 'new')
```

Comment:
`state` is an attribute for "&lt;target&gt;". Here it seems you set it on &lt;trans-unit&gt; .
But I think we do not need to set state=new as a missing &lt;target&gt; has the same effect